proposals/NNNN-hlsl-loop-unroll-factor.md

```diff
@@ -0,0 +1,38 @@
+# HLSL Loop Unroll Factor
+
```

Comment:
I need a proposal number.

Reply:
Process is that you pick this _just_ before completing the, PR based on what's in main at the time.

Reply:
0022